packages/react-reconciler/src/ReactFiberReconciler.js

```diff
@@ -109,6 +110,13 @@ type DevToolsConfig = {|
   // This API is unfortunately RN-specific.
   // TODO: Change it to accept Fiber instead and type it properly.
   getInspectorDataForViewTag?: (tag: number) => Object,
+  // Used by RN in-app inspector.
+  getInspectorDataForViewAtPoint?: (
```

Comment:
I don't think either of these two should be part of this type because they're not actually used by the core DevTools protocol. It's kind of an adhoc RN specific addition.

Instead we should make this a renderer specific object.

```
rendererConfig: RendererInspectionConfig
```

Then you can define the type for RendererInspectionConfig through the HostConfig mechanism. So that each renderer gets their own type. That can then be safely referenced in the react-native folder.

So Fabric and RN host configs gets this type:

```
export type RendererInspectionConfig = {
  getInspectorDataForViewTag?: (tag: number) => Object,
  getInspectorDataForViewAtPoint?: (...) => ...;
};
```

You then import that from here using:

```
import type {RendererInspectionConfig} from './ReactFiberHostConfig';
```

Reply:
I like this suggestion.